Safewaan/HonkedIn
# Create Events Page Unit Tests

## Implementation description
Test Additions
* Add test to verify events are loaded on first render
* Add test to verify event titles are displayed
* Add test to verify event dates are displayed
* Add test to verify participants and total participants are displayed
* Add test to display "Active" status only if the event is active
* Add test to display "Cancelled" status only if the event is cancelled

Code Changes
* Updated event pages to use local timezone instead of GMT



## Steps to test
1. run `npm test`. Verify all unit tests pass.



## What should reviewers focus on?
* Are there other tests that should be added? Do the current tests correctly test the front-end?
